packages/atomic/src/components/search/atomic-notification/atomic-notifications.new.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import {renderComponent} from '@coveo/atomic/storybookUtils/render-component';
+import {wrapInSearchInterface} from '@coveo/atomic/storybookUtils/search-interface-wrapper';
+import type {Meta, StoryObj as Story} from '@storybook/web-components';
+
+const {decorator, play} = wrapInSearchInterface({
+  search: {
+    preprocessSearchResponseMiddleware: (response) => {
+      response.body.triggers = [
+        {
+          type: 'notify',
+          content:
+            'This is a demo notification. It contains text that may span over a different number of lines depending on your screen width. Notifications are returned by the Coveo Search API.',
+        },
+        {
+          type: 'notify',
+          content:
+            'This is a different notification. Any amount of notifications can be returned by the Coveo Search API.',
+        },
+      ];
+      return response;
+    },
+  },
+});
+
+const meta: Meta = {
+  component: 'atomic-notifications',
+  title: 'Atomic/Notification',
+  id: 'atomic-notifications',
+
+  render: renderComponent,
+  decorators: [decorator],
+  parameters: {
+    controls: {expanded: true, hideNoControlsWarning: true},
+  },
+  play,
+};
+
+export default meta;
+
+export const Default: Story = {
+  name: 'atomic-notifications',
+  args: {},
```

Comment:
Why do we sometimes have this empty args ?
Can it be removed ?

Reply:
Mh yes 👍 
- [ ] Remove empty args